scripts/run.sh

```diff
@@ -1,16 +1,160 @@
 #!/bin/bash
```

Comment:
Sure... I had some issues with 2 spaces due to having the indented `cat<-EOF` command. I'll unindent that and switch to spaces. :+1: